library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadAdapter.kt

```diff
@@ -226,10 +240,182 @@ internal sealed class TransactionPayloadViewHolder(view: View) : RecyclerView.Vi
             const val LUMINANCE_THRESHOLD = 0.25
         }
     }
+
+    internal class BodyJsonViewHolder(
```

Comment:
Can we be consistent with the naming here?
This should be a `BodyLineCollapsibleViewHolder`